src/classes/PanelData.js

```diff
@@ -166,6 +166,18 @@ class PanelData {
 	 * @return {Object}		panel data shared by multiple views
 	 */
 	get panelView() {
+		let currentTheme = this._confData.get('currentTheme');
+		let theme;
+		if (currentTheme !== 'default') {
+			theme = (this._confData.get('themes') || {})[currentTheme];
```

Comment:
@zarembsky this should now be `theme = this._confData.get('themes')[currentTheme];` right?